cms/static/js/certificates/models/certificate.js

```diff
@@ -0,0 +1,71 @@
+// Backbone.js Application Model: Certificate
+
+define([
+    'backbone', 'js/certificates/models/signatory', 'js/certificates/collections/signatories', 'underscore', 'underscore.string', 'gettext', 'backbone-relational', 'backbone.associations', 'coffee/src/main'
+],
+function(Backbone, Signatory, SignatoryCollection, _, str, gettext) {
+    'use strict';
+    _.str = str;
```

Comment:
I took a look at this and it seems Backbone is looking for the _.str varialble -- @asadiqbal08 , can you provide further explanation?